recipes/cctz/all/conanfile.py

```diff
@@ -20,53 +21,65 @@ class CCTZConan(ConanFile):
     }
     default_options = {
         "fPIC": True,
-        "shared": False, 
-        "build_tools": False
+        "shared": False,
+        "build_tools": True
     }
 
-    _source_subfolder = "source_subfolder"
-    _build_subfolder = "build_subfolder"
+    exports_sources = ["CMakeLists.txt", "patches/**"]
+    generators = "cmake"
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
 
     def config_options(self):
         if self.settings.os == "Windows":
             del self.options.fPIC
 
     def configure(self):
-        if self.settings.os == "Windows" and \
-           self.settings.compiler == "Visual Studio" and \
+        if self.options.shared:
+            del self.options.fPIC
+
+    def validate(self):
+        if self.settings.compiler == "Visual Studio" and \
            tools.Version(self.settings.compiler.version) < 14:
-           raise ConanInvalidConfiguration("CCTZ requires MSVC >= 14")
+            raise ConanInvalidConfiguration("CCTZ requires MSVC >= 14")
```

Comment:
```suggestion
               raise ConanInvalidConfiguration("CCTZ requires MSVC >= 14")
```

Reply:
?

Reply:
The whitespace indent is off... right?

Reply:
I've fixed indentation, your suggestion breaks it

Reply:
Ahhhhhh thank you for your patients... looking the the while file I seen the line break in the if above so yes... sorry the one spacebar looked out of place in the diff